Dockerfile

```
@@ -1,9 +1,13 @@
-FROM ubuntu
+FROM  debian 
+RUN echo "deb http://files.freeswitch.org/repo/deb/debian/ jessie main" > /etc/apt/sources.list.d/99FreeSWITCH.test.list
 RUN apt-get update
-RUN apt-get upgrade -y
 RUN apt-get install -y --no-install-recommends build-essential autoconf automake libtool zlib1g-dev libjpeg-dev libncurses-dev libssl-dev libcurl4-openssl-dev python-dev libexpat-dev libtiff-dev libx11-dev wget git
-RUN git clone -b v1.2.stable https://freeswitch.org/stash/scm/fs/freeswitch.git /usr/local/src/freeswitch
-RUN cd /usr/local/src/freeswitch; ./bootstrap.sh -j
-RUN cd /usr/local/src/freeswitch; ./configure --prefix=/opt/freeswitch
-RUN cd /usr/local/src/freeswitch; make; make install
-RUN cd /usr/local/src/freeswitch; make all cd-sounds-install cd-moh-install
+RUN wget -O - http://files.freeswitch.org/repo/deb/debian/key.gpg |apt-key add -
+RUN apt-get update
+RUN DEBIAN_FRONTEND=none APT_LISTCHANGES_FRONTEND=none apt-get install -y --force-yes freeswitch-video-deps-most
+RUN git config --global pull.rebase true
+RUN git clone https://freeswitch.org/stash/scm/fs/freeswitch.git freeswitch.git
+RUN cd freeswitch.git; ./bootstrap.sh -j; perl  -i -pe 's/#applications\/mod_av/applications\/mod_av/g' modules.conf
+RUN  cd freeswitch.git; ./configure --prefix=/opt/freeswitch ; make; make install cd-sounds-install cd-moh-install samples; 
+
```

Use `&&` instead of `;` to fail early.